### PR TITLE
add fossasia projects : Meilix and Loklak 

### LINKED ISSUE #222: More FOSSASIA Projects should be added
**Actual Behaviour**

Only SUSI

**Expected Behaviour**

All should be there

**Steps to reproduce it**

Add:
- [ ] Badgeyay
- [ ] Loklak
- [ ] Meilix
- [x] Susper
- [ ] Phimp.me
- [ ] Eventyay
- [ ] Yaydoc
- [ ] PSLab

**Screenshots of the issue**

screenshot 2018-11-05 at 1 50 50 pm

**Would you like to work on the issue?**

Yes, open to all(also)

### PR DESCRIPTION
**Pull request by pythongiant**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://pythongiant.github.io/gci18.fossasia.org/)

- Fixes #222 

## Description / Changes

Add Loklak and Meilix to Fossasia Project sections

## Screenshots if any:
screensho22

- - - - - - - - - - - -

### CODE REVIEW

**Comment by pythongiant** [comment]
ok @fragm3 will not do from next time

File: _includes/footer.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <footer class="footer-1 bg-dark">
   <div class="container">
     <div class="row">
+
```
Comment (liveHarshit) [inline _includes/footer.html]: Why these changes?

File: _includes/footer.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <footer class="footer-1 bg-dark">
   <div class="container">
     <div class="row">
+    
+
```
Comment (liveHarshit) [inline _includes/footer.html]: same

File: _includes/footer.html (hunk 1 above)
Comment (liveHarshit) [reply _includes/footer.html]: Do not commit this file.

File: components.html [hunk 3]
```diff
@@ -164,7 +164,10 @@ <h2> Students </h2>
                         <div class="row mb80 mb-xs-0">
                             <div class="col-md-8 col-md-offset-2 text-center">
                                 <h1>Connect with Fossasia on Social Media</h1>
+
```
Comment (liveHarshit) [inline components.html]: WTC? Also, remove this file from the commit.

Comment (tstreamDOTh) [reply components.html]: @liveHarshit The intend of Code In & especially for this site is to promote learning
